app/src/main/java/org/eyeseetea/malariacare/domain/entity/Survey.java

```diff
@@ -36,6 +38,9 @@ public void setStatus(int status) {
     }
 
     public SurveyAnsweredRatio getSurveyAnsweredRatio() {
```

Comment:
to get SurveyAnsweredRatio we should not use the survey model but GetSurveyAnsweredRatioUseCase